### DISCUSSION TITLE
Gradients does not work without From

### DISCUSSION THREAD

**Discussion by jenstornell** [tailwindlabs/tailwindcss#2228] [Help] [answered]
I don't know if it's a bug or not. From is required but the others are optional. Can't the From value be "magical" when not present just like the To value?

### Works

```
from via to
from via
from to
from
```

### Does not work

```
via to
via
to
```

To try the combinations out you can use https://tailwind-gradient-designer.csspost.com/

### Without From but others included

![tailwind-gradient-designer csspost com_ (2)](https://user-images.githubusercontent.com/927011/90973961-3a21cb80-e527-11ea-8d6f-e4db11d2c69e.png)

### With From without the others

![tailwind-gradient-designer csspost com_ (1)](https://user-images.githubusercontent.com/927011/90973963-3d1cbc00-e527-11ea-948d-ba2a2281c4ef.png)

**Comment by adamwathan** [comment, marked as answer]
Yep this is expected behavior and in the documentation. I couldn't think of a clever way to make it work but maybe I missed one. Recommendation is to just flip the gradient and use from instead of to 👍🏻

Super cool tool btw, nice work! 🙌🏻
